docs/userguide/parameters.md

```diff
@@ -127,11 +136,43 @@ SubstanceParameter(
 )
 ```
 
-The ``encoding`` option defines what kind of descriptors are calculated:
-* ``MORDRED``: 2D descriptors from the [Mordred package](https://mordred-descriptor.github.io/documentation/master/).
-  Since the original package is now unmaintained, baybe requires the community replacement [mordredcommunity](https://github.com/JacksonBurns/mordred-community)
-* ``RDKIT``: 2D descriptors from the [RDKit package](https://www.rdkit.org/)
-* ``MORGAN_FP``: Morgan fingerprints calculated with RDKit (1024 bits, radius 4)
+The ``encoding`` option defines what kind of descriptors are calculated using the
+[scikit-fingerprints] package.
+Any of the fingerprint classes provided by the package can be used.
+The encoding can be specified either by passing the corresponding [`SubstanceEncoding`] member
+(click to see full list of options) or its string representation, e.g. use 
```

Comment:
Isn't this redundant? I guess that this will be formatted properly as a link

Reply:
Yes, true. If you think it's redundant, feel free to remove. I didn't have it in my first version but that thought it might not be obvious to people that they will actually also find the explicit list of possible values there.

Reply:
would prefer a reference where users can see all valid choices, this is always good

Reply:
Ok, then it's already two people who think mentioning explicitly where to find the full list doesn't hurt. Let's keep it as it is then